### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by nolan-cerebras** [ndey96/SYDE-Survey#1]
- [x] Timestamp
- [x] What gender do you identify with?
- [x] What year were you born in?
- [x] Which ethnic background(s) do you identify with?
- [x] What most closely describes your religious school of thought?
- [x] How would you describe your political leanings?
- [x] What city did you grow up in?
- [x] Were you born in Canada?
- [x] Which of the following accelerated programs, if any, did you graduate from?
- [x] What was your entrance average from high-school (top 6 grades only)?
- [x] Which of the following extracurricular activities did you do in high school, if any?
- [x] Before starting university, what industry did you see yourself working in?
- [x] What university faculties did you apply to?
- [x] How often did you attend lectures this term? 🤓
- [x] How stressful was this term? 😫
- [x] Did you find a co-op job this term?
- [x] How much did you pay for rent? Format as amount, currency, rate. For example: 600, CAD, monthly
- [x] How long was your commute to school in minutes?
- [x] How stressful was this term? 😫.1
- [x] How did you find this job?
- [x] What was your job title? (Cleaning)
- [x] In which city did you work? (Cleaning) {GTA, KW, Other Canada, Bay Area, Other USA/International, Seattle}
- [x] Which industry best describes the company you worked for?
- [x] What size was the company when you worked there?
- [x] What was your Jobmine performance rating?
- [x] What was your base salary (not including benefits)? Format as "Amount, Currency, Rate". 
- [x] Was housing or a housing stipend provided? If so, what was the estimated value? Format as "Amount, Currency, Rate". (OMIT)
- [x] How much did you pay for rent? Format as amount, currency, rate. For example: 600, CAD, monthly.1
- [x] How long was your commute to work in minutes?
- [x] Did your FYDP group consist solely of SYDE students?
- [x] What domain most accurately describes your FYDP?
- [x] Are you proud of your FYDP?
- [x] Do you or your group plan to pursue your FYDP further beyond 4B?
- [x] How did you find out about SYDE?
- [x] When did you join the SYDE 2019 cohort?
- [x] How many years did it take to finish your degree?
- [x] When (if at all) did you go on exchange?
- [x] If you went on exchange, what university did you attend? (cleaning)
- [x] How satisfied were you with the design courses and projects?
- [x] If you could go back in time, would you still do SYDE?
- [x] If you answered "No" to the above question. What would you do instead?
- [x] What the most interesting course you took? (cleaning)
- [x] What the most useful course you took? (cleaning)
- [x] What CSEs did you take? (cleaning)
- [x] What TEs did you take? (cleaning)
- [x] What was your term average in 1A?
- [x] What was your term average in 1B?
- [x] What was your term average in 2A?
- [x] What was your term average in 2B?
- [x] What was your term average in 3A?
- [x] What was your term average in 3B?
- [x] What was your term average in 4A?
- [x] What was your term average in 4B?
- [x] What was your parents' combined income at the time you entered university?
- [x] What is the highest level of education achieved by either of your parents?
- [x] Have either of your parents studied/worked in a STEM-related field?
- [x] Were either of your parents born in North America?
- [x] Approximately how much of your total university expenses were funded by your family? 
- [x] How much debt are you graduating with?
- [x] How much do you have in savings?
- [x] Of your 5 closest friends you've made in university, how many are in SYDE?
- [x] If you were not part of the original SYDE 2019 cohort, do you feel like the class has integrated you into the 2019 community?
- [x] How many friends do you have in the SYDE 2019 class?
- [x] Out of 8 academic terms, how many of them did you attend at least one student-organized SYDE social event?
- [x] Which of the following inter-generational SYDE activities did you participate in, if any?
- [x] How many upper years are you still in contact with?
- [x] How many lower years are you still in contact with?
- [x] Which, if any, do you plan on donating to as an alumni?
- [x] Did you do any of the following during university?
- [x] How would you rate your cooking ability before university? (histogram of differences)
- [x] How would you rate your cooking ability after university? (histogram of differences)
- [x] How would you rate your fitness level before university? (histogram of differences)
- [x] How would you rate your fitness level after university? (histogram of differences)
- [x] Which of the following is the most important factor for you when considering a job/career?
- [x] What was your favourite food establishment on/near campus throughout the past 5 years? (cleaning)
- [x] If you had to offer advice to a first year, what would it be? (cleaning) (just pick 3-5 that are representative)
- [x] What do you expect to be doing a year from now?
- [x] What will be your job title? (cleaning)
- [x] Which industry best describes the company you will be working at?
- [x] When did you receive the job offer?
- [x] What city will you be working in? (cleaning)
- [x] Do you consider this city to be within your hometown / region?
- [x] Will you be working for a company you previously worked for during one of your co-ops?
- [x] What is your base salary compensation? (wrap into TC)
- [x] What is your equity compensation, if any? (wrap into TC)(assume /4)
- [x] What is your signing and/or relocation bonus, if any? (wrap into TC)
- [x] What is your commission or yearly bonus, if any? (wrap into TC)
- [x] Does your job require you to relocate outside Canada?
- [x] Why did you take a job outside Canada?
- [x] Do you plan to come back to Canada?
- [x] When do you plan to return?
- [x] If you plan to return or not return, why?
- [x] 10 years from now, which industry do you hope to be working in?
- [x] 10 years from now, how many people from our class do you think you'll be in touch with?
- [x] In the next 10 years do you plan to pursue any of the following forms of higher education?
- [x] If you intend to get married, at what age do you expect to do so?
- [x] If you intend to have children, at what age do you expect to do so?
- [x] Where in the world would you like to settle down? (cleaning)
- [x] What sexuality do you identify with?
- [x] In your FINAL year at school, on average how often did you drink socially?
- [x] In your FIRST year at school, on average how often did you drink socially?
- [x] In your FINAL year at school, on average how often did you consume marijuana?
- [x] In your FIRST year at school, on average how often did you consume marijuana?
- [x] Which of the following recreational drugs did you try for the first time at university? 
- [x] Have you ever engaged in any of the following?
- [x] Have you failed a course or term in SYDE?
- [x] Have you experienced or been diagnosed with any of the following mental health concerns?
- [x] With regards to anxiety and/or depression, what was the source(s) of distress?
- [x] Have you sought any of the following forms of mental health support? 
- [x] How many relationships did you have during university?
- [x] How did you meet your significant others?
- [x] How much of your university career, in months, was spent in a relationship?
- [x] Did you have sex before beginning university?
- [x] Did you have sex for the first time in university?
- [x] How many unique sexual partners have you had in university, if any?

**Comment by iFallUpHill** [comment]
Cleaning Checklist (For me)


- [x] What was your job title? (Cleaning)
- [x] In which city did you work? (Cleaning) {GTA, KW, Other Canada, Bay Area, Other USA/International, Seattle}
- [x] If you went on exchange, what university did you attend? (cleaning)
- [x] What the most interesting course you took? (cleaning)
- [x] What the most useful course you took? (cleaning)
- [x] What CSEs did you take? (cleaning)
- [x] What TEs did you take? (cleaning)
- [x] What was your favourite food establishment on/near campus throughout the past 5 years? (cleaning)
- [x] If you had to offer advice to a first year, what would it be? (cleaning) (just pick 3-5 that are representative)
- [x] What will be your job title? (cleaning)
- [x] What city will you be working in? (cleaning)
- [x] Where in the world would you like to settle down? (cleaning)
- [x] What is your base salary compensation? (wrap into TC)
- [x] What is your equity compensation, if any? (wrap into TC)(assume /4)
- [x] What is your signing and/or relocation bonus, if any? (wrap into TC)
- [x] What is your commission or yearly bonus, if any? (wrap into TC)

**Comment by nolan-cerebras** [comment]
Additional graphs that require multiple columns

- [x] wage vs ethnicity
- [x] Wage vs gender
- [x] relationships vs gender
- [x] sex vs gender
- [x] wage vs parental income
- [x] wage vs marks
- [x] lecture attendance vs grades
- [x] Wage vs gender density graph
- [x] relationships vs gender density graph
- [x] sex vs gender density graph

**Comment by nolan-cerebras** [comment]
Things left to do:
- [x] Update nolan's website
- [x] Fix graphs
- [x] add in text data
- [x] organize slides by section
- [x] change color for each section
- [x] Find a class photo
- [x] Write slide blurbs
- [x] Write foreword/afterword stuff
- [x] title all of the slides
- [x] sort demographics
- [x] sort coop
- [x] sort school
- [x] sort social lifestyle
- [x] sort the future
- [x] update content slide with finalized sections
- [x] proofread
- [x] make a rollout strategy (maybe ask a past person)
- [x] release full version online
- [x] send SYDE department partial version

**Comment by iFallUpHill** [comment]
- [x] add first year quotes / why moving back / why moved for job

**Comment by nolan-cerebras** [comment]
Booom! https://www.reddit.com/r/uwaterloo/comments/dn09d2/uwaterloo_systems_design_engineering_2019_class/
